pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "pre-commit-hooks"
-version = "0.7.0"
```

Comment:
The latest tag is already `0.7.4` maybe we should go directly to `0.7.5` - and then tag to be in sync again?